tools/bridge/bridge/config.py

```diff
@@ -19,8 +19,8 @@ def validate_rpc_url(url: Any) -> None:
 def validate_positive_integer(number: Any) -> float:
     if not isinstance(number, int):
         raise ValueError(f"{number} is not an integer")
-    if number <= 0:
-        raise ValueError(f"{number} must be positive")
+    if number < 0:
```

Comment:
that's not a positive integer anymore. Please rename the function, e.g. `validate_non_negative_integer`.

Reply:
I learned that the mathematicians can't decide between both definitions, but I don't care so I apply it. :laughing:

Reply:
:heavy_check_mark:

Reply:
> I learned that the mathematicians can't decide between both definitions, but I don't care so I apply it. laughing

zero has never been a positive number. Please stop spamming github issues.